Update download-tools-nuget.md

antoinesaliba opened the pull request:
The current documentation contains a script located here https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/download-tools-nuget?view=op-9-1 that fails with the following error:

`./nuget : Unable to find package 'Microsoft.CrmSdk.XrmTooling.PluginRegistrationTool'`

To resolve this issue, `https://api.nuget.org/v3/index.json` needs to be added as a source to nuget. This PR updates the documentation to include this step.

### Related Issues
https://github.com/MicrosoftDocs/dynamics-365-customer-engagement/issues/2467
https://github.com/MicrosoftDocs/dynamics-365-customer-engagement/issues/2437
https://github.com/MicrosoftDocs/dynamics-365-customer-engagement/issues/2419

JimDaly commented:
@antoinesaliba 
Thanks for your contribution.

When I run the script with the changes proposed, I get the following error:

    The name specified has already been added to the list of available package sources. Please provide a unique name.

It isn't a fatal error, the tools download successfully. This would be an improvement for those who do not have this source already.

Seems like the success of the script depends on what sources are already configured.
Do you get the error above the second time you run the script with your changes?

I'm thinking perhaps we should have a check and only add the source if it doesn't exist.
Looking at https://stackoverflow.com/questions/52843248/check-if-a-nuget-source-is-already-configured
It seems like we need something like this:

```
if((./nuget source | ? { $_ -like "https://api.nuget.org/v3/index.json"}))
{
   .\nuget sources Add -Name nuget.org -Source  https://api.nuget.org/v3/index.json
}
```

Is this something you could factor in?

antoinesaliba commented:
Thank you for the speedy response @JimDaly!

Yes, you're absolutely right, I get the same error if I try to re-run the script with this change. I've updated the PR with the suggestion you made [97b9355](https://github.com/MicrosoftDocs/dynamics-365-customer-engagement/pull/2641/commits/97b9355dd60508604cc942bc31d5c04be8c27cd0). I also updated the source name to include the version just to be extra clear since there is a v2 as well.
